alerta/utils/api.py

```diff
@@ -87,16 +87,16 @@ def process_alert(alert: Alert) -> Alert:
     return alert
 
 
-def process_action(alert: Alert, action: str, text: str) -> Tuple[Alert, str, str]:
-
+def process_action(alert: Alert, action: str, text: str, time_out: int) -> Tuple[Alert, str, str, int]:
+    #named variable time_out so doesnt conflict with kwargs name
```

Comment:
I don't understand why this is called "time_out" and not "timeout". Why would it conflict with `kwargs` name? 😕

Reply:
it just looked funny to me when it was timeout=timeout, but i guess there is not technical reason for it.